docs/pages/cloud/downloads.mdx

```diff
@@ -51,6 +51,33 @@ Linux Debian Packages              | [amd64](https://get.gravitational.com/telep
 Linux RPM Packages                 | [amd64](https://get.gravitational.com/teleport-ent-(=cloud.version=)-1.x86_64.rpm) | [arm64](https://get.gravitational.com/teleport-ent-(=cloud.version=)-1.arm64.rpm) | [armv7](https://get.gravitational.com/teleport-ent-(=cloud.version=)-1.arm.rpm) | [i386](https://get.gravitational.com/teleport-ent-(=cloud.version=)-1.i386.rpm) |
 macOS Packages                     | [amd64](https://get.gravitational.com/teleport-ent-(=cloud.version=).pkg) |
 
+## Teleport package repositories
+
+Teleport currently maintains DEB repositories for Debian compatible systems.
+
```

Comment:
I think we should include a short sentence explaining that you would follow these commands to add a DEB repository and install Teleport.

It would also be great if we could list the different repositories we offer, e.g., that we have on repository per OS version/major Teleport version. Otherwise, users following the instructions  may not be aware of the way we separate and categorize our release channels.

Also, are there any OS versions/Teleport versions we don't have repositories for (especially ones a user might expect us to support)?